src/check-nsfw.js

```diff
@@ -87,6 +88,7 @@ async function checkNSFW(message, urls) {
 
 			// check if the prediction is black listed
 			if (classification.className === 'Porn' || classification.className === 'Hentai' || classification.className === 'Sexy') {
+				if (classification.probability < 0.8) return; // if low confidence then do nothing
```

Comment:
Also here, should probably check the confidence before the type to save a bit of time. It doesn't matter what the type is if it isn't confident

Reply:
Fixed!